src/utils.js

```diff
@@ -95,7 +95,7 @@ function contextify(context, request) {
   return request
     .split('!')
     .map((r) => {
-      const splitPath = r.split('?', 2);
+      const splitPath = r.split('?');
```

Comment:
It should be more smart, what is we will have 2-3-4-5-more loaders?

Reply:
Aren't multiple loaders handled by `.split('!')` ?

Reply:
This only removes the assumption that the loader syntax is limited to `"loader?options"` but can also be `"loader?options?ref"`. I'm not sure if there can be more options, so therefore I just removed the limit completely.